docs/advanced-topics/hybrid-models.md

```diff
@@ -0,0 +1,137 @@
+# Hybrid model deployment
+
+FHE enables cloud applications to process private user data without running the risk of data leaks. Furthermore, deploying ML models in the cloud is advantageous as it eases model updates, allows to scale to large numbers of users by using large amounts of compute power, and protects model IP by keeping the model on a trusted server instead of the client device.
+
+However, not all applications can be easily converted to FHE computation and the computation cost of FHE may make a full conversion exceed latency requirements.
+
+Hybrid models are a compromise between on-premise or on-device deployment and full cloud deployment. Hybrid deployment means parts of the model are executed on the client side and parts are executed in FHE on the server side. Concrete ML supports hybrid deployment of neural network models such as MLP, CNN and Large Language-Models.
+
+{% hint style="warning" %}
+If model IP protection is important, care must be taken in choosing the parts of a model to be executed on the cloud. Some
+black-box model stealing attacks rely on knowledge distillation
+or on differential methods. As a general rule, the difficulty
+to steal a machine learning model is proportional to the size of the model, in terms of numbers of parameters and model depth.
+{% endhint %}
```

Comment:
I am not sure what users should do with that information. Should we point them to the litterature maybe?

Reply:
I wanted a "disclaimer" - simply making a model hybrid doesn't fix all problems.

Reply:
I would maybe link to some literature on KD attacks indeed !

Reply:
Maybe worth creating an issue and properly documenting this in a second time?